### ISSUE TITLE
Need to clarify which  gcc-arm-embedded version for compiling

### ISSUE THREAD

**Issue by luc-github** [aethaniel/ExperimentalCore-sam#54]
Hi when use latest gcc-arm-none-eabi-5_3-2016q1-20160330-win32.exe from  https://launchpad.net/gcc-arm-embedded
the compilation failed :

```
error: conflicting types for 'utoa'   
 extern char* utoa( unsigned long value, char *string, int radix );
...
stdlib.h:201:8: note: previous declaration of 'utoa' was here
 char * _EXFUN(utoa,(unsigned, char *, int));
```

if I use previous version gcc-arm-none-eabi-4_9-2015q3-20150921-win32.exe
I got different warnings (same as travis-ci output) but compilation is Ok

so need to freeze  gcc-arm-embedded version or modify makefile to support latest version of gcc-arm-embedded

**Comment by luc-github** [comment]
May be an easy solution to ensure right version is used is to add a download script with hard coded versions to download like they do for smoothieware : https://github.com/Smoothieware/Smoothieware/blob/edge/win_install.cmd

but it means to add all necessary tools also (at least for window) 

Just my 2 cents

**Comment by aethaniel** [comment]
indeed, ARM GCC releases from this year bring utoa in newlib. The GCC provided with Arduino IDE should work well, being the current reference.
